### DISCUSSION TITLE
PyPI Download statistics

### DISCUSSION THREAD

**Discussion by blaylockbk** [blaylockbk/Herbie#363] [Show and tell]
Something I don't understand is why [PyPI Download Statistics](https://pypistats.org/packages/herbie-data) is reporting so many downloads in the last week.

![image](https://github.com/user-attachments/assets/97f516b5-67fa-4a18-9688-a871ec80c8a8)

What does this mean? Certainly Herbie is not suddenly used by so many new users. Perhaps someone is using Herbie in some cloud computing workflows?

**Comment by williamhobbs** [comment]
I recall someone involved with https://github.com/pvlib/pvlib-python once mentioning that PyPi stats are not very helpful for figuring out how popular a repo is with real users because of automated downloads from PyPi (mirrors? bots? I can't remember what the source is...). 

They have ended up using GitHub contributors, GitHub stars, and discussion forum registration (I think for the Google mailing list) as an indicator of community growth/engagement. 

http://dx.doi.org/10.21105/joss.05994

Either way, the recent bump is probably a good sign for popularity, though!

**Comment by blaylockbk** [comment]
I'm still amazed when I look at pypi download statistics. 

Some power user recently switch from Python 3.11 to 3.12
![image](https://github.com/user-attachments/assets/607b9dd0-f8b2-43a4-8a8f-545ae879186a)
